### ISSUE TITLE
"asdf" not flagged

### ISSUE THREAD

**Issue by JVimes** [streetsidesoftware/vscode-spell-checker#2027]
It seems to think "asdf" is a word. That's what I use to verify checking is working. Is this on purpose?

![image](https://user-images.githubusercontent.com/6148201/173932023-b4fbfedc-b71e-4f82-bafe-1845a7702f59.png)

**Comment by Jason3S** [comment]
To find the source of a word, the following command (assuming you have Node JS installed) will show where it comes from:

```sh
npx cspell trace asdf
```
<img width="202" alt="image" src="https://user-images.githubusercontent.com/3740137/173934888-a6b57158-8083-4e56-8cdc-9174bcbea4c9.png">

It was added here: https://github.com/streetsidesoftware/cspell-dicts/pull/689 because of this tool https://asdf-vm.com/

## Removing `softwareTerms` dictionary

You can remove the `softwareTerms` dictionary using the following:

### **VS Code Settings**

<img width="958" alt="image" src="https://user-images.githubusercontent.com/3740137/173935505-a3ee0c6a-e99f-4208-bb22-e513403b0087.png">

or

```json
"cSpell.dictionaries": ["!softwareTerms"]
```

### **cspell.json**
```json
{
  "dictionaries": ["!softwareTerms"]
}
```

## Block `asdf`

### **VS Code Settings**

<img width="354" alt="image" src="https://user-images.githubusercontent.com/3740137/173936059-30e69433-f386-4fec-8419-c772b535bf80.png">

or 

```json
"cSpell.flagWords": ["asdf"]
```

### **cspell.json**
```json
{
  "flagWords": ["asdf"]
}
```

**Comment by JVimes** [comment]
Aha, thank you. I really appreciate it.
